GsaGH/Components/0_Model/SaveGsaModel.cs

```diff
@@ -105,16 +105,16 @@ public class SaveGsaModel : GH_OasysDropDownComponent {
     }
 
     internal void Save(ref GsaModel model, string fileNameAndPath) {
-      if (!fileNameAndPath.EndsWith(".gwb")) {
-        fileNameAndPath += ".gwb";
+      if (!fileNameAndPath.EndsWith(".gwa") && !fileNameAndPath.EndsWith(".gwa")) {
+        fileNameAndPath += ".gwa";
```

Comment:
since gwa is a lot slower let´s keep 
`fileNameAndPath += ".gwa";`
here

Reply:
ok